packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+meta:
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary for the submission process using the OIN Manager.
+
+**What you need**
+
+* a built and tested OIN integration that is ready for Okta verification
+
+---
+
+## Overview
+
+Prior to using the [OIN Manager](https://oinmanager.okta.com) to submit your OIN integration, you need to review the following guidelines to prepare for the artifacts requested during the submission process:
+
+* [Logo guidelines](#logo-guidelines)
+* [App description guidelines](#app-description-guidelines)
+* [Use case guidelines](#use-case-guidelines)
+* [Test account guidelines](#test-account-guidelines)
+* [Customer configuration document guidelines](#customer-configuration-document-guidelines)
+
+Before you follow the submission guidelines, ensure that your integration does not contain [unsupported OIN features](#unsupported-oin-features).
+
+## OIN limitations
+
+Integrations with the following Okta features can't be published in the OIN catalog:
+
+> **SWA apps:** SWA app integrations are no longer accepted for publication in the OIN catalog. However, existing SWA apps are still maintained by the OIN team.
+
+> **Custom authorization server:** To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server), including the default server. You can only use the [Org authorization server](/docs/concepts/auth-servers/#org-authorization-server).
+
+> **Refresh token:**  The `refresh_token` option isn't supported for apps published in OIN.
+
+> **Unsupported scopes:** <br>
+> * `offline_access` scope isn't available since refresh tokens aren't supported for apps published in OIN.
+> * `groups` claims as described in the supported [Scopes](/docs/reference/api/oidc/#scopes) section isn't supported.
+
+<ApiAmProdWarning />
+
+## Logo guidelines
+
+The app logo that you submit to the OIN Manager in the **General Settings** > **App icon** field must conform to the following guidelines:
+
+* The logo file size must be less than one MB.
+* The recommended logo dimension has a 1:1 (square) aspect ratio.
+
+   > **Note:** OIN app icons are placed in a 200 x 200 pixels grid.
+
+* The recommended logo contains only a logomark and not a logotype (such as a company icon and not the company name).
+
+   > **Note:** The company name is already listed in plain text in the OIN catalog, so adding a company name image is redundant.
+
+* Avoid logos with both an icon and a wordmark.
```

Comment:
This is the same as the above?

I think we should remove "recommended" / "avoid" and make them requirements.

* The logo must be a square icon (200 x 200 recommended) and not contain the company name
> **Note:** The name of the application will be displayed alongside the icon

Reply:
What if we put some screenshots or links to places where the logo is displayed so they can see it in context?

Reply:
Sorry, I just read your Jira comment after going through all this where you suggest the same.